terraform-use-modules/finish.md

```diff
@@ -0,0 +1,7 @@
+# Congratulations!
+
+You've reached the end of the scenario!
+
+If you'd like to learn more, try one of these resources on HashiCorp Learn:
```

Comment:
Looks like there's only one link - either add more or update CTA.

Reply:
Updated it to include AWS provider auto